src/tests-bns/event-server-tests.ts

```diff
@@ -1,3 +1,4 @@
+import * as fs from 'fs';
```

Comment:
I don't think this is required anymore (same with imports below).
You should revert this file to be like `develop`:
```
git checkout develop -- src/tests-bns/event-server-tests.ts
```